$NetBSD: patch-aa,v 1.2 2015/12/29 23:34:43 dholland Exp $

Remove setmode() declaration that is different from BSD's. Caused
build failure on Interix.

--- awka/fin.h.orig	2001-07-06 01:03:46.000000000 -0400
+++ awka/fin.h
@@ -51,6 +51,4 @@ unsigned PROTO ( fillbuff, (int, char *,
 
 extern  FIN  *main_fin ;  /* for the main input stream */
 void   PROTO( open_main, (void) ) ;
-
-void  PROTO(setmode, (int,int)) ;
 #endif  /* FIN_H */
